src/tools/kata-ctl/src/types.rs

```diff
@@ -33,3 +34,28 @@ pub struct CheckItem<'a> {
     pub fp: BuiltinCmdFp,
     pub perm: PermissionType,
 }
+
+// Builtin module parameter check handler type
+pub type BuiltinModuleParamFp = fn(args: &str) -> bool;
```

Comment:
I think this would be better as:

```suggestion
pub type BuiltinModuleParamFp = fn(args: &str) -> Result<()>;
```

Since a `Result` is a boolean, we can check for `Ok())` to mean `true` and Err(e)` to mean `false`. And if the predicate function fails, we can display that error to the user.